templates/mocktail/show.html.twig

```diff
@@ -1,126 +1,60 @@
-{% extends 'base.html.twig' %} {% block title %} {{ parent() }} {#
-{{ mocktail.name }} #} {% endblock %} {% block stylesheet %}
-<link rel="stylesheet" href="assets/styles/pages/mocktail_show.css" />
-<link
-    rel="stylesheet"
-    href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
-/>
-{% endblock %} {% block header %}
-<div class="page-heading header-text">
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-12">
-                <h1>{{ mocktail.name }}</h1>
-            </div>
-        </div>
-    </div>
-</div>
+{% extends 'base.html.twig' %} {% block title %}
+{{ parent() }}
+{{ mocktail.name }} - Mocktail Details {% endblock %} {% block stylesheet %}
+<link rel="stylesheet" href="assets/styles/pages/mocktail-show.css" />
 {% endblock %} {% block content %}
-<div class="section title-detail-page">
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-4">
-                <div class="section-heading">
-                    <a href="/mocktails"> Retour</a>/{{ mocktail.name }}
+<main>
+    <article class="mocktail-details section">
```

Comment:
the article tag should wrap all the moktail informations. then you can split all components into sections rather than in articles if you want